charts/rancher-vsphere-csi/rancher-vsphere-csi/100.0.0/values.yaml

```diff
@@ -25,7 +25,7 @@ csiController:
     enabled: false
```

Comment:
Old asset should get deleted and index needs updating.

Reply:
@cbron the version isn't changing so the index.yaml doesn't need to be modified. The asset also got modified, which means that it did change